core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -732,7 +732,8 @@ class DAGScheduler(
         job.jobId, callSite.shortForm, partitions.length))
       logInfo("Final stage: " + finalStage + "(" + finalStage.name + ")")
       logInfo("Parents of final stage: " + finalStage.parents)
-      logInfo("Missing parents: " + getMissingParentStages(finalStage))
+      val missingStages = getMissingParentStages(finalStage)
+      logInfo("Missing parents: " + missingStages)
```

Comment:
Super minor but should this use string interpolation? as a general principle for speed and conciseness, though it won't matter here.